src/meson.build

```diff
@@ -40,6 +40,7 @@ libfwupdprivate = static_library(
     'fu-progressbar.c',
     'fu-quirks.c',
     'fu-smbios.c',
+    'fu-systemd.c',
```

Comment:
Given these symbols are only used with a systemd enabled compile, shouldn't this file be dynamic?

Reply:
Good idea, fixed.